.github/workflows/avm.res.managed-identity.user-assigned-identity.yml

```diff
@@ -0,0 +1,81 @@
+name: "avm.res.managed-identity.user-assigned-identity"
```

Comment:
@elanzel as discussed offline please also make sure of the following:
- update codeowners file to uncomment the line referencing your module
- make sure your module owner team is properly added to the intended parent team CC: @matebarabas